pghistory/core.py

```diff
@@ -237,10 +237,10 @@ def _generate_history_field(tracked_model, field):
     """
     field = tracked_model._meta.get_field(field)
 
-    if isinstance(field, models.AutoField):
-        return models.IntegerField()
-    elif isinstance(field, models.BigAutoField):  # pragma: no cover
+    if isinstance(field, models.BigAutoField):
```

Comment:
Looks like there was no test for it before. So either we can write the test now, or add back the `# pragma: no cover`.